bin/update_pythons.py

```diff
@@ -213,10 +217,7 @@ def __init__(self) -> None:
 
         self.macos_6 = CPythonVersions(plat_arch="macosx_x86_64", file_ident="macosx10.6.pkg")
         self.macos_9 = CPythonVersions(plat_arch="macosx_x86_64", file_ident="macosx10.9.pkg")
-        self.macos_u2 = CPythonVersions(
-            plat_arch="macosx_universal2",
-            file_ident="macos11.0.pkg",
-        )
+        self.macos_u2 = CPythonVersions(plat_arch="macosx_universal2", file_ident="macos11.0.pkg")
```

Comment:
This is the "magic" comma in Black, which lets you force multiline. I didn't want to do that here, so I removed it.

PS: I originally tried to write this without Black, but the first time a ran I had 20 or so Flake8 errors; I had no interest in waisting time trying to fix them by hand, so I ran `black bin/update_python.py`, and they all went away. So this is our one Black'ed file. :)